js/main.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+var photosCount = 25;
```

Comment:
Тут и ниже - константы
```suggestion
var PHOTOS_COUNT = 25;
```